src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -469,8 +470,8 @@ public void verifyUserLoggedIn(final String userName) {
         driver.switchTo().frame("PreviewFrame");
       }
       // open nav if on mercury, required to see login data
-      if (driver.findElements(MERCURY_NAV_ICON).size() > 0) {
-        wait.forElementClickable(MERCURY_NAV_ICON);
+      if (driver.findElements(MERCURY_SKIN).size() > 0) {
```

Comment:
I got the impression you would like to wait a bit if Mercury skin is not loaded yet - in here, implicit wait is 0 and explicit is not applied at all, is this your intention?